conans/client/conan_api.py

```diff
@@ -877,10 +877,12 @@ def search_packages(self, reference, query=None, remote_name=None, outdated=Fals
             if remote_ref.ordered_packages:
                 for package_id, properties in remote_ref.ordered_packages.items():
                     package_recipe_hash = properties.get("recipe_hash", None)
+                    # Artifactory uses field 'requires', conan_center 'full_requires'
+                    requires = properties.get("requires", []) or properties.get("full_requires", [])
```

Comment:
Ok, but knowing that ``requires`` is the ``zlib/1.X.Y`` converted version, used for the package_id hash. This probably requires normalization in the server side too?

Reply:
`requires` from Artifactory contains the full package reference (there is no `full_requires` field), Conan Server `full_requires` contain the full package reference (there is no `requires` field).

Reply:
Oh! Ok then.